lbz/lambdas/client.py

```diff
@@ -63,3 +65,31 @@ def invoke(
         error_message = f"Error response from {function_name} Lambda (op: {op}): {lambda_result}"
         logger.error(error_message, extra=dict(data=data, response=response))
         return response
+
+    @classmethod
+    def request(
+        cls,
+        function_name: str,
+        path: str,
+        method: str,
+        params: dict = None,
+        query_params: dict = None,
+        body: dict = None,
+        headers: dict = None,
+    ) -> Response:
+        data = APIGatewayEvent(
+            resource_path=path,
+            method=method,
+            path_params=params,
+            query_params=query_params,
+            body=body,
+            headers=headers,
+        )
+        raw_response = client.lambda_.invoke(
+            FunctionName=function_name,
+            Payload=json.dumps(data, cls=cls.json_encoder).encode("utf-8"),
+            InvocationType="RequestResponse",
+        )
+        response: dict = json.loads(raw_response["Payload"].read().decode("utf-8"))
```

Comment:
What about the try-except that is implemented above? Maybe we should make a protected method for invoking the Lambda Function? 🤔